qa/rpc-tests/listtransactions.py

```diff
@@ -32,6 +32,11 @@ def check_array_result(object_array, to_match, expected):
 
 class ListTransactionsTest(BitcoinTestFramework):
 
+    def setup_nodes(self):
+        #This test requires mocktime
+        enable_mocktime()
+        return start_nodes(4, self.options.tmpdir)
+
```

Comment:
For this test I agree that something is needed to fix the test to work with this PR.  Enabling mocktime as you do does the trick; alternatively, it would also be sufficient to insert `self.nodes[0].generate(1)` at line 36 below, as that would cause the nodes to leave IBD.

I don't feel strongly about which workaround is better (mocktime or generating a block to leave IBD), just mentioning it in case anyone else has an opinion.